src/plone/z3cform/fieldsets/README.rst

```diff
@@ -146,7 +146,7 @@ With this in place, let's update the form once again::
 At this point, we should have a set of default fields that represent the
 ones set in the adapter::
 
-  >>> form.fields.keys()
+  >>> list(form.fields.keys())
```

Comment:
alternatively, we can use `[k for k in form.fields.keys()]` which is more explicit about it being an iterator.

Reply:
Since I saw the `list()` multiple times in other doctests of core packages I used it here also ... but I don't mind.